prometheus/src/request.tsx

```diff
@@ -16,28 +16,31 @@ export async function isPrometheusInstalled() {
   return [false, null, null];
 }
 
-export function fetchMetrics(
-  prefix: string,
-  query: string,
-  from: string,
-  to: string,
-  step: string
-) {
+export function fetchMetrics(data: {
+  prefix: string;
+  query: string;
+  from: number;
+  to: number;
+  step: number;
+}) {
   const params = new URLSearchParams();
-  if (from) {
-    params.append('start', from);
+  if (data.from) {
+    params.append('start', `${data.from}`);
```

Comment:
Use `.toString()` instead of the literals here and in the other examples. Literals should be used for string formatting.